api/locator.go

```diff
@@ -34,4 +34,6 @@ type Locator interface {
 	Fill(value string, opts goja.Value)
 	// Focus on the element using locator's selector with strict mode on.
 	Focus(opts goja.Value)
+	// GetAttribute of the element using locator's selector with strict mode on.
```

Comment:
This is a late comment, but we mention "strict mode" a lot in this file, but it's never explained what that really means. Initially I confused it with JS strict mode, which is entirely different, and it wasn't clear until I dug into `injected_script.js`.

Can you maybe expand the `Locator` comment, or at the first occurrence explain what strict mode is? No need to have it repeated everywhere, but it would be helpful to mention it once.

Reply:
Please see: 9fd4fc1